R/extract.R

```diff
@@ -195,25 +197,42 @@ extract_preprocessor.workflow <- function(x, ...) {
 
 #' @rdname extract-workflow
 #' @export
-extract_postprocessor.workflow <- function(x, estimated = TRUE, ...) {
+extract_postprocessor.workflow <- function(x, ..., estimated = FALSE) {
+  if (has_postprocessor_tailor(x)) {
+    return(extract_tailor_workflow(x, estimated = estimated))
+  }
+
+  cli_abort("The workflow does not have a postprocessor.")
+}
+
+# TODO this still needs a generic in hardat
+extract_tailor_workflow <- function(x, ..., estimated = TRUE) {
+  check_dots_empty()
+
   if (!is_bool(estimated)) {
     cli_abort(
       "{.arg estimated} must be a single {.code TRUE} or {.code FALSE}."
     )
   }
+  if (!has_postprocessor_tailor(x)) {
+    cli_abort("The workflow must have a tailor postprocessor.")
+  }
```

Comment:
this code is currently unreachable because `has_postprocessor_tailor()` is called to guard this call. Will be fine to keep if we expect `extract_tailor_workflow()` to be used other places

Reply:
Yes, the idea is to turn this into `extract_tailor.workflow()` which can be used directly.